bootstrap/src/index.ts

```diff
@@ -48,7 +49,40 @@ const withLogger: Middleware = async (execute) => async (input: AdapterRequest)
   }
 }
 
-const middleware = [withLogger, skipOnError(withCache), withStatusCode]
+const withMetrics: Middleware = async (execute) => async (input: AdapterRequest) => {
+  const recordMetrics = () => {
+    const labels: Parameters<typeof metrics.httpRequestsTotal.labels>[0] = {
+      method: 'POST',
+    }
+    const end = metrics.httpRequestDurationSeconds.startTimer()
+
+    return (statusCode?: number, type?: metrics.HttpRequestType) => {
+      labels.type = type
+      labels.statusCode = metrics.normalizeStatusCode(statusCode)
+      end()
+      metrics.httpRequestsTotal.labels(labels).inc()
+    }
+  }
+
+  const record = recordMetrics()
+  try {
+    const result = await execute(input)
+    record(
+      result.statusCode,
+      result.data.maxAge
```

Comment:
Thought `maxAge` was included in data, but is included at the top level, at `result.maxAge`

Reply:
Ill check both, just incase